nist-interface/src/App.js

```diff
@@ -6,12 +6,13 @@ import {
 
 import './App.css';
 import CssBaseline from '@mui/material/CssBaseline';
-import { Dashboard } from './components/dashboard';
+import { Dashboard } from './pages/dashboard';
```

Comment:
`Dashboard` isn't being used, is it?

Reply:
So, yes will do cleanups for not needed imports.